src/app/component/AddressInput/AddressInput.jsx

```diff
@@ -31,34 +26,35 @@ const AddressInput = (props) => {
     if (value && placesRef?.current) {
       placesRef.current.setVal(value);
     }
-  }, [placesRef, value]);
+  }, [placesRef, value, disabled]);
 
   return (
     <div style={error ? { border: "solid red" } : {}}>
-      <AlgoliaPlaces
-        placeholder={placeholder || "Search address"}
-        name={stage}
-        options={{
-          appId: ALGOLIA_APP_ID,
-          apiKey: ALGOLIA_API_KEY,
-          language: "en",
-          countries: ["us"], // we have to support more countries in the future
-          // Other options from https://community.algolia.com/places/documentation.html#options
-        }}
-        onChange={(query) => handleLocation(query)}
-        onLimit={({ message }) => message && console.log(message)}
-        onClear={onClear}
-        placesRef={setRef}
-      />
+      {disabled ? (
+        <input className="ap-input" disabled="true" value={value} />
```

Comment:
When the component doesn't have a disabled so you just steal the styles and conditionally render a disabled input. Works flawlessly lolol

Reply:
hahahaha hahahaahaha what the ?